src/index.html

```diff
@@ -21,7 +21,441 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logotip MOYO"
```

Comment:
The alt attribute should describe the image. 'logotip MOYO' doesn't give a clear description of what the image is. Consider changing it to something more descriptive like 'MOYO Logo'.